source/hic_hal/freescale/k20dx/gpio.c

```diff
@@ -57,24 +57,36 @@ void gpio_init(void)
     PIN_nRESET_GPIO->PDDR &= ~PIN_nRESET;
     PIN_nRESET_PORT->PCR[PIN_nRESET_BIT] = PORT_PCR_MUX(1);
 
-    // Keep powered off in bootloader mode
-    // to prevent the target from effecting the state
-    // of the reset line / reset button
     if (!daplink_is_bootloader()) {
```

Comment:
Since the code in this conditional isn't turning on power it doesn't need to be disabled for the bootloader.

Reply:
Ok, will change.

Generally I was trying to keep things as close to the original as possible to reduce risk. But I'm happy to clean it up if you're comfortable. (Of course, you have the board farm, so that helps ;)